python/lsst/meas/extensions/scarlet/scarletDeblendTask.py

```diff
@@ -674,7 +679,8 @@ def deblend(self, mExposure, catalog):
             idFactory.notify(maxId)
 
         filters = mExposure.filters
-        self.log.info(f"Deblending {len(catalog)} sources in {len(mExposure)} exposure bands")
+        self.log.info("Deblending %d sources in %d exposure bands", len(catalog), len(mExposure))
+        logTime = time.time()
```

Comment:
maybe call this `lastLoggedTime`? (or see later comment about changing it to next time a log message is needed)